themes/hugo-lithium/static/css/main.css

```diff
@@ -366,3 +366,14 @@ code {
     box-shadow: none;
   }
 }
+
+
+/* bring quotes coser to previous paragraph */
+blockquote {
+  margin-block-start: -1em;
+}
+
+/* bring bullets closer to previous paragraph */
+ul, ol { 
+  margin-top: -1em;
```

Comment:
Same comment as above. I would change to `margin-top: -10px;`

I would also add `margin-bottom: 25px;`, looks like the margin after is currently not consistent with paragraphs and probably should be.

Reply:
combined this with the above style for consistency